pkg/policy/source.go

```diff
@@ -34,6 +35,13 @@ func DetectPolicy(name string, subPolicy string) (*Policy, bool, error) {
 	}, true, nil
 }
 
+func classifyError(detectorType string, err error) error {
+	if (detectorType == "file") && strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
use `errors.Is(err, fs.ErrNotExist)`

Reply:
Ah, I tried it and it doesn't work as the original error has been formatted multiple times once it gets to this line.
I tried unwrapping it to get the original, but that doesn't work as well, since `go-getter` doesn't use the `%w` operand.